### ISSUE TITLE
`check-values` rejects any license value I tried

### ISSUE THREAD

**Issue by Antonius-S** [gajus/eslint-plugin-jsdoc#783]
## Expected behavior

No complaints at
```js
 /**
 * @module test
 * @license MIT
 */
'use strict';
```

## Actual behavior

`Invalid JSDoc @license: "MIT
"; expected SPDX expression: https://spdx.org/licenses/.`

## ESLint Config

```js
// Format JS (or JSON) code here
"extends": [
        "eslint:recommended",
        "plugin:jsdoc/recommended"
    ],
```

## ESLint sample

```js
// Format JS code here
 /**
 * @module test
 * @license MIT
 */
'use strict';
```

## Environment

- Node version: v12.22.6
- ESLint version v7.32.0
- `eslint-plugin-jsdoc` version: 36.0.8

Only `{"licensePattern":".*"}` solved the issue

**Comment by brettz9** [comment]
I'm not able to replicate. Are you on Windows?

**Comment by Antonius-S** [comment]
That was a catch! Thanks!

**Comment by brettz9** [comment]
Thanks for the clear report. A release should be in the pipeline shortly.
